operator/Makefile

```diff
@@ -103,6 +103,10 @@ vet:
 generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths="./..."
 
+# Generate Clientset
+create-client: test
+	./hack/update-codegen.sh	
```

Comment:
Doesn't `update-codegen.sh` just download the generator tools?

Reply:
And create the client folder.

Reply:
Got it! Thanks for clarifying that.